### ISSUE TITLE
[Suggestion] Pack icons as squares rather than circles.

### ISSUE THREAD

**Issue by Seth0067** [gorilla-devs/GDLauncher#436]
First - great, professional job on the launcher so far.  Feels like a modern app rather than something i would have used six years ago.  Your efforts are applauded.

Second (more to the point), while I feel the desire to do something a little different is generally awesome, having modpack icons fit within a circular frame sometimes conflicts with the existing icon designs.  This forces a crop that is sometimes fine, but often clips text or info, making the icon look awkward.  

Keep up the great work!

**Comment by TOLoneWolf** [comment]
This is currently what the next branch has which is a rectangle instead of a circle:
https://cdn.discordapp.com/attachments/539914597591679008/699620529291657257/unknown.png

I would say this is what your suggesting?

**Comment by Seth0067** [comment]
More or less, but the art is still cropped, which is what the problem is, right?  Are the text overlays seen on mouseover only or is that how all the instance icons view as?  The Pack title is definitely fighting with the art background.

**Comment by TOLoneWolf** [comment]
yes text overlays are visable without the mouse hovering over them. 
On mouse over the work Play overlays above that in the center.
![image](https://user-images.githubusercontent.com/8376628/80546735-1507bc80-897c-11ea-9ef0-bdc5ce5244ab.png)

**Comment by AzureDoom** [comment]
If you feel like, you want to use this in the main branch, @CrankySupertoon and myself have been working on https://github.com/CrankySupertoon/GDLauncher-Classic which is a fixed up version of the main branch which features a more Next look and feel to it. Currently the released version is behind a bit, hopefully Cranky pushes a new build here soon.

**Comment by Seth0067** [comment]
Thanks AzureDoom, I have no issues using the current version, just wanted to pass along some suggestions to improve the launcher.  Thanks for the heads up though!

**Comment by kreezxil** [comment]
maybe there should be option global config to change the silhouette masks, like circles, diamonds, squares, trapezoid, stars, w/e

**Comment by furtabs** [comment]
I do kinda agree with this.

**Comment by blarfoon** [comment]
Is this still an issue?

**Comment by Seth0067** [comment]
Sorry for the late reply, wanted to take some time to review latest build.

This is still an issue, even though the entire design of this has been iterated on at least twice by now (great to see you guys still doing this).  Now that i've had a chance to see the latest presentation (1.0.5), the cover art for the modpacks is presented in a darkened/desaturated state in all presentations (local instances, browse for new instance, instance detail) so as to contrast text which is overlaid on the art directly, Unfortunately, the new designs are still not serving the art or the modpacks very well.

This is all about creating a pleasant browsing experience - both the user and the creators want to see the "cover art" in a good light.  The user wants to see the art clearly - to identify known packs quickly or be attracted to new ones.  The creators also want to see the fruits of their work presented clearly (this is their brand). 

Succinctly, I would reconsider the design decision to use text-on-art, and for simplicity, and clarity, feature the pack art at the closest resolution possible to the pack artwork as it was created (presumably for Twitch), within the square aspect ratio.

**Comment by furtabs** [comment]
I agree with this totally. Its like getting the github logo 
![image](https://user-images.githubusercontent.com/48419040/94875081-aeeafe00-0421-11eb-9996-f7795413e680.png)

and cuttings Octocat's head off. Well it is.
![image](https://user-images.githubusercontent.com/48419040/94875111-cc1fcc80-0421-11eb-9edd-ff0098b7b549.png)

**Comment by BigAl607** [comment]
Happens on custom packs when you change the icon. Top, bottom & sides get cropped, and I can't find anywhere that says what the dimensions of the image should be
